microcosm_pubsub/dispatcher.py

```diff
@@ -105,19 +127,24 @@ def invoke_handler(self, handler, media_type, content):
             )
             return handler_with_context(content)
         except SkipMessage as skipped:
-            extra = self.sqs_message_context(content)
-            extra.update(skipped.extra)
+            self.opaque.update(skipped.extra)
             logger.info(
                 "Skipping message for reason: {}".format(str(skipped)),
-                extra=extra,
+                extra=self.opaque.as_dict(),
```

Comment:
I am going to work on the logging code next. I'm not clear whether `extra=` requires a mapping or a dict.